Swiftcord/Views/Message/MessageStickerView.swift

```diff
@@ -0,0 +1,203 @@
+//
+//  MessageStickerView.swift
+//  Swiftcord
+//
+//  Created by Vincent Kwok on 23/2/22.
+//
+
+import SwiftUI
+import Lottie
+import DiscordKitCore
+import DiscordKit
+import CachedAsyncImage
+
+
```

Comment:
There are too many blank lines here, please ensure there's only one blank line at most to pass the linting rule.